setup.py

```diff
@@ -56,8 +55,8 @@
         'pyyaml',
         'requests',
         'scipy>=1.6',
-        'scitools-iris>=3.4.0',
-        'shapely[vectorized]',
+        'scitools-iris>=3.6.0',
+        'shapely',
         'stratify',
         'yamale',
     ],
```

Comment:
did you forget to add the pip-installable ESMValTool sample data?

Reply:
I don't quite understand. Could you explain in more detail what you mean with this comment?

Reply:
the sample data PyPI package that you removed from environmen.yml, man